packages/backend/userServices.js

```diff
@@ -178,15 +178,62 @@ async function findAllReviews() {
 // function to find reviews by userId
 async function findReviewsByWrittenBy(userId) {
```

Comment:
Is it possible to rename this function? I'm a bit confused by the purpose

Reply:
Yes it is possible. I didn't make or change this function though, but I guess I'll look into changing the name.